### DISCUSSION TITLE
blog/how-i-completed-my-first-hacktoberfest/

### DISCUSSION THREAD

**Discussion by giscus** [AnilSeervi/Blog#8] [Comments]
# blog/how-i-completed-my-first-hacktoberfest/

Here's how I completed my first Hacktoberfest.

https://anil.gatsbyjs.io/blog/how-i-completed-my-first-hacktoberfest/

**Comment by AnilSeervi** [comment]
Test Giscus Comments

**Reply by AnilSeervi** [reply]
test reverse comment from Github Discussion
